Core/include/Acts/Utilities/VectorHelpers.hpp

```diff
@@ -253,4 +254,23 @@ inline std::pair<double, double> incidentAngles(
   return {phi, theta};
 }
 
+/// Calculate the deltaR between two vectors.
+/// @note DeltaR is defined as sqrt(deltaPhi^2 + deltaEta^2)
+/// @tparam Derived Eigen derived concrete type
+/// @param v1 First vector
+/// @param v2 Second vector
+/// @return The deltaR value
+template <typename Derived>
+double deltaR(const Eigen::MatrixBase<Derived>& v1,
+              const Eigen::MatrixBase<Derived>& v2)
+  requires(Eigen::MatrixBase<Derived>::RowsAtCompileTime == 3)
+{
+  double dphi = abs(phi(v1) - phi(v2));
+  if (dphi > std::numbers::pi) {
+    dphi = std::numbers::pi * 2 - dphi;
+  }
```

Comment:
there is some utility in `periodic.hpp` to do angle difference: like `difference_periodic(phi1, phi2, 2pi)`